### PR TITLE
EOS-25118: Make path of cortx.conf file path configurable

### PR DESCRIPTION
**Pull request by vastradparayya**
# Problem Statement
- make path of cortx.conf file path configurable
- As of now, cortx.conf file is in /etc/cortx path. This path needs to be made configurable, we need to read the path present in cluster.conf, under local key.

# Design
-  https://jts.seagate.com/browse/EOS-25118

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: N
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N] NA
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/common/common.py [hunk 1]
```diff
@@ -9,9 +9,21 @@ class CortxConf:
...
+        Conf.load(CortxConf._index, \
+            f"json://{os.path.join(local_storage_path, 'utils/conf/cortx.conf')}", \
             skip_reload=True)
 
+    @staticmethod
+    def get_storage_path(key):
+        """ Get the config file path """
+        Conf.load('cluster', 'yaml:///etc/cortx/cluster.conf', skip_reload=True)
+        return Conf.get('cluster', f'cortx>common>storage>{key}')
+        # if not path:
```
Comment (sachinpunadikar) [inline py-utils/src/utils/common/common.py:22]: Raise error if path is None / empty

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on the comment

File: py-utils/src/utils/common/common.py (hunk 1 above)
Comment (vastradparayya) [reply py-utils/src/utils/common/common.py]: Done.

**Review by selvakumaar5496 — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]

**Review by rohit-k-dwivedi — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**